src/tests/Parser/Brewmaster/T20_2pc.test.js

```diff
@@ -0,0 +1,18 @@
+import T20_2pc from 'Parser/BrewmasterMonk/Modules/Items/T20_2pc';
+import { events, processEvents } from './Fixtures/SimpleFight';
+
+const item = new T20_2pc();
+
+describe('Brewmaster.T20_2pc', () => {
+  beforeAll(() => {
+    processEvents(events, item);
+  });
+  it('How many orbs spawned', () => {
+    expect(item.orbTriggeredBy2Pc).toBe(1);
+  });
+  it('How many brews were used which could spawn an orb', () => {
```

Comment:
This is used to calculate the actual proc rate of the T20 2pc, each brew has a 40% chance